src/ethereum_clis/cli_types.py

```diff
@@ -212,6 +213,23 @@ class TransitionToolInput(CamelModel):
     env: Environment
 
 
+class TransitionToolConfig(CamelModel):
+    """Transition tool config."""
+
+    fork: Fork = Field(..., alias="network")
+    chain_id: int = Field(..., alias="chainid")
+    blob_schedule: BlobSchedule
```

Comment:
This one is very similar to the already-existing `TransitionToolContext`, so we can just repurpose that one instead.